community/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,123 @@
+As a community, we want to encourage diversity in all aspects, and set a standard
+for respectful and appropriate conduct across all communication channels. For
+this purpose, we have created this code of conduct, which serves as a guide for
+interaction with the SourceCred community. It is adopted from [several well known](#license)
+Code of Conducts of open source projects.
+
+## People
+
+In the following sections:
+
+ - **we/our**: refers to the members of the SourceCred community, who exist on equal footing.
+ - **SourceCred**: refers to the general project or community.
+ - **Benevolent Dictator**: refers to @decentralion
```

Comment:
I think of myself as 'Temporary Benevolent Dictator'. We could future proof this document by having a role like 'Community Maintainer' who is in charge of maintaining the SC community as a healthy, respectful, and inclusive space. Then (as TBD) I can also be de-facto community maintainer until we choose someone else.

We can also just keep Benevolent Dictator in, and just update the doc once we change structures. Curious to hear others' thoughts.

Reply:
I'm +1 for keeping it in until we've officially changed it. My reasoning is that "Temporary Benevolent Dictator" doesn't inspire confidence, and "Community Maintainer" isn't a role that we've fully decided on or flushed out.

Reply:
While I agree "Community Maintainer" isn't well defined, when I read this as a future community member, "Temporary Benevolent Dictator" comes off a bit harsh and unwelcoming. The role of dictator in the popular imagination is not one that cares about your feelings, or one you would go to about a sensitive issue (to the cage with you dissident!). It totally makes sense on Discourse posts explaining what the TBD is. It's being real about a sensitive power issue in a way that comes off as light and humorous. But in this context, I would suggest something lighter. "Community Maintainer" is not one I've heard before, but conveys the point OK. "Community Manager" is a fairly common term. "Facilitator"?

Reply:
You're right that unfamiliarity with BD or TBD can have it come across as a bit harsh - I'm used to Guido from Python being referred to as the "Benevolent Dictator" so I don't have this bias - he has been an important, positive force. I'm happy to change the term here to some consensus. that others can reach.

Reply:
If we're exploring the language space with terms that are both values aligned and in usage, I'd offer something along the lines of *Servant Leader* as it happens our Temporary Benevolent Dictator practices Servant Leadership (in my opinion).
- https://en.wikipedia.org/wiki/Servant_leadership 

The community Manager and Facilitator terms come off a bit toothless. In my opinion the upside of Benevolent Dictator (especially those of us with prior positive associations) implied both the alignment towards good, and the power to act. Of course, dictator level power has the obvious association with corruption. The *Temporary* added to Benevolent Dictator helps but doesn't solve the issue of the negative connotation toward unchecked power. Another consideration would be uniqueness. Facilitator and community manager are good in that the imply non-uniqueness (of course at the cost of implied less power to act). Servant Leader does necessarily imply uniqueness, though to hammer home the point you could have a servant leadership council, but of course that starts to feel bureaucratic.

It might be worth opening a discourse thread on the topic of world choice around the roles of maintainers, and community leaders. This is one place where word choice can directly effect norms.

Reply:
I'm not familiar with Servant Leader, but anything in that family of "Servant" (or to take from SLURM terms) "Master" and "Slave" I would personally prefer to stay away from, if possible. I definitely feel weird calling @decentralion that. I do see your point about not having teeth - do you really think teeth are needed here? Some of the best leaders / supervisors I have had have championed their roles through kindness. Kindness, empathy, compassion, are much more admirable qualities in a leader that would encourage others to listen. Maybe if the code of conduct document itself is "the thing that has teeth" we don't need to imply via word choice that a particular individual does?

Reply:
I'm with @vsoch in being a bit uncomfortable with "Servant" here.

I like using the word "maintainer"; for me (coming from open-source) maintainers have authority over the domain that they are maintaining, and can do things like block changes, remove problematic contributors, etc. Since the community values are something we care about, I propose the term "Values Maintainer" here.

Reply:
I like that too - it's relatively straight forward and lines up well with open source. I just pushed a change so folks can read and see if they like it too.